proto/messages.proto

```diff
@@ -218,5 +218,5 @@ enum ClientAction {
 }
 
 message Calling {
-  required string content = 1;
+  required bytes content = 1;
```

Comment:
There are clients out there which can receive this message though.

The scenario I want to avoid is that existing public clients today will receive a `Calling` message with `bytes` but then throw an error since they expect `string`. But if you are sure they will handle that in a appropriate way we could of course leave it as it is.